src/adapter/src/coord/appends.rs

```diff
@@ -390,7 +391,14 @@ impl Coordinator {
 
                         // Notify the external clients of the result.
                         for response in responses {
-                            let (ctx, result) = response.finalize();
+                            let (mut ctx, result) = response.finalize();
+                            if ctx.session().vars().transaction_isolation()
+                                == &IsolationLevel::StrongSessionSerializable
+                            {
+                                ctx.session_mut()
+                                    .local_timestamp_oracle()
+                                    .apply_write(timestamp);
+                            }
```

Comment:
Not covered by any test case

Reply:
This is due to the split between the non-shareable and shareable timestamp oracles. This code is only run when the shareable timestamp oracle is used, but it looks like that isn't used in SLTs. Do you know which kinds of tests have been used to test that feature? (CC @aljoscha maybe he knows).

Reply:
Hmm, I'd think the postgres oracle is enabled here: 
https://github.com/MaterializeInc/materialize/blob/975cbad891ace301a27bf2f0347e953404372c74/misc/python/materialize/mzcompose/__init__.py#L92

Which should make its way through: 
https://github.com/MaterializeInc/materialize/blob/4f3516718ebea22ef39b6c4c65880204abb2bc21/misc/python/materialize/mzcompose/services/sql_logic_test.py#L33

Then here:
https://github.com/MaterializeInc/materialize/blob/0107a2f8efa74433e35b38663ddbf202a3822799/src/sqllogictest/src/runner.rs#L1787

And finally here:
https://github.com/MaterializeInc/materialize/blob/0107a2f8efa74433e35b38663ddbf202a3822799/src/sqllogictest/src/runner.rs#L1065

There is logging that shows when the postgres oracle is enabled. Do we have that?

Reply:
Oh, if that's the case then I think this is just a false negative.

Reply:
That's definitively possible, the coverage is unfortunately not always 100% accurate.